presto-docs/src/main/sphinx/release/release-0.232.rst

```diff
@@ -0,0 +1,52 @@
+* Add ``query_max_total_memory_per_node`` and ``query_max_memory_per_node`` session properties.
+* Add support to show whether functions have variable arity in ``SHOW FUNCTIONS``.
+* Add support to show whether functions are built-in in ``SHOW FUNCTIONS``.
+* Add ``use_exact_partitioning`` session property that forces repartitioning if repartitioning is possible.
+* Add support for ``ALTER FUNCTION``.
+* Add configuration property ``resource-groups.reload-refresh-interval`` to control the frequency of reloading resource group information from the database. The default value is 10 seconds.
+* Add support for using the Thrift protocol to shuffle data. This can be configured using the configuration property internal-communication.task-communication-protocol. Possible values are HTTP or Thrift.
+* Add support for using the Thrift protocol to announce node state. This can be configured using the configuration property internal-communication.server-info-communication-protocol. Possible values are HTTP or Thrift.
+* Add session property ``list_built_in_functions_only`` to support hiding user-defined SQL functions in ``SHOW FUNCTIONS``.
+* Add ``tdigest_agg``, ``merge(tdigest)``, ``value_at_quantile(tdigest, quantile)``, ``values_at_quantiles(tdigest, quantiles)``, ``quantile_at_value(tdigest, quantile)``, ``quantiles_at_values(tdigest, quantile)`` for creating, merging, and querying t-digests.
+* Upgrade airlift/slice to 0.38.
+
+Hive Changes
+____________
+* Fix an issue where queries could fail with buffer overflow when writing ORC files.
+* Add support for handling statistics to the Alluxio metastore.
+* Add Alluxio metastore which connects to the ``Alluxio catalog service`` (https://docs.alluxio.io/os/user/2.1/en/core-services/Catalog.html).
+* Add session property ``shuffle_partitioned_columns_for_table_write`` to make Presto shuffle data on the partition columns before writing to partitioned unbucketed Hive tables.  This increases the maximum number of partitions that can be written in a single query by a factor of the total number of writing workers.  The property is false by default. (:pr:`14010`).
+* Expose Hive table properties via system table$properties table.
+* Change error code from ``HIVE_METASTORE_ERROR`` to ``HIVE_TABLE_DROPPED_DURING_QUERY`` when a ``DROP TABLE`` query fails due to another query dropping the table before this query has finished.
+* Upgrade Alluxio version from 2.1.1 to 2.1.2.
+
+Kudu Changes
+_____________
+* Add KERBEROS authentication.
+
+Kafka changes
+_____________
+* Update KAFKA connector to 2.3.1, which improves implementation and performance (:pr:`13709`).
```

Comment:
````
Improve ``Kafka`` connector implementation and performance (:pr:`13709`)
```

Reply:
@rschlussel commented to make it capitalized

Reply:
sorry, I meant capitalize the first letter since it's a proper noun.